frontend/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Why this line was removed?

Reply:
Prettier keeps deleting it automatically even its in ignore

Reply:
Not fully related to your comment but some people decided to gitignore this file to avoid dealing with changes on this file for git, because it is generated at every build.

https://github.com/vercel/next.js/issues/26533